dashboard/app/views/levels/editors/fields/_authored_hints.haml

```diff
@@ -67,6 +67,15 @@
             'markdown',
             {
               callback: function(editor, change) {
+                //Looking for `.markdown_preview div --> div --> [XML Blockly divs]` structure
```

Comment:
nit: Recommend some slight styling changes.

From this:
```
                //Looking for `.markdown_preview div --> div --> [XML Blockly divs]` structure
                //gets .markdown_preview div
                let parent = space.find('.markdown_preview').get(0); 
```
to this:
```
                // Looking for `.markdown_preview div --> div --> [XML Blockly divs]` structure
                // gets .markdown_preview div.
                let parent = space.find('.markdown_preview').get(0); 

```

(with a blank line before the next statement)